### PR TITLE
mq - dozer

### PR DESCRIPTION
**Pull request by MirlindQeriqi**
Mapper applied in ProgramService
ProgramTranslator deleted

### CODE REVIEW

**Review by mosgjig — APPROVED** [review]

**Comment by mosgjig** [comment]
Did you run your service test?  they should be broken, and will have to mock the mapper now.

**Comment by mosgjig** [comment]
I closed it on accident.  Please address the unit tests for service and re-open.

**Comment by MirlindQeriqi** [comment]
Yeah, I didn't run the service test. And now that I did I can see they're broken. Lemme work on it and I'll commit the changes

File: src/main/java/org/prnhs/javaee/swim/services/config/MapperConfig.java [hunk 1]
```diff
@@ -37,8 +39,10 @@ protected void configure() {
                         .fields(PASSWORD, PASSWORD, oneWay())
                         .fields(MY_LIST, MY_LIST, collectionStrategy(Boolean.TRUE, RelationshipType.NON_CUMULATIVE))
                         .fields(MY_USERS, MY_USERS, collectionStrategy(Boolean.FALSE, RelationshipType.NON_CUMULATIVE));
-                
+                mapping(Program.class, ProgramDto.class)
+                        .fields(field("id"), field("key"), oneWay())
```
Comment (mosgjig) [inline src/main/java/org/prnhs/javaee/swim/services/config/MapperConfig.java]: use constants please :)

**Review by mosgjig — CHANGES_REQUESTED** [review]

**Review by mosgjig — APPROVED** [review]

**Pull request merged**